src/Features/Core/Portable/PullMemberUp/MemberAnalysisResult.cs

```diff
@@ -19,16 +19,37 @@ namespace Microsoft.CodeAnalysis.PullMemberUp
         /// </summary>
         public readonly bool ChangeOriginalToNonStatic;
 
+        /// <summary>
+        /// Indicate whether this member's declaration in destination needs to be made to abstract. It is only used by the dialog UI.
+        /// If this property is true, then pull a member up to a class will only generate a abstract declaration in the destination.
+        /// It will always be false if the refactoring is trigger from Quick Action.
+        /// </summary>
+        public readonly bool MakeMemberDeclarationAbstract;
+
+        /// <summary>
+        /// Indicate whether pulling this member up would change the destination to abstract. It will be true if:
+        /// 1. Pull an abstract member to a non-abstract class
+        /// 2. The 'Make abstract' check box of a member is checked, and the destination is a non-abstract class
```

Comment:
note: i would like a quick discussion with other IDE members if we need to support this case.  Seems super specialized and potentially unneccesary.